manifests/operator.yaml

```diff
@@ -534,7 +534,7 @@ spec:
       priorityClassName: gmp-critical
       containers:
       - name: operator
-        image: gke.gcr.io/prometheus-engine/operator:v0.9.0-gke.1
+        image: gke.gcr.io/prometheus-engine/operator:v0.10.0-gke.6
```

Comment:
I doubt this works due to https://github.com/GoogleCloudPlatform/prometheus-engine/issues/1013
 
(with readOnlyRootFilesystem: true)

Have you tested it?

Reply:
Ah yes - indeed https://github.com/GoogleCloudPlatform/prometheus-engine/pull/944 breaks this.

@bernot-dev or @TheSpiritXIII do you know if we have an image published we could use that works with that change? Presumably HEAD worked (`v0.13.0-rc.0`).

OTOH maybe trying to "fix" our nightly manifest in this way isn't a great idea and I should abandon the PR as it isn't possible to always have a working manifest in HEAD anyway due to GKE build pipeline delays...

Reply:
I mean, you are not making it worse so let's go =D

Good point that we could have issues going forward, but we have same problem with AM and Prometheus and for those we HAVE to make them workable in atomic way otherwise tests won't pass. It's just with PE images we are "allowed" to cheat a bit.

I wonder if splitting manifests to OSS official ones VS source for managed GMP should be different things e.g. for GMP we could simply use helm templates directly and have different values (: We could actually get rid of kustomize (or move kustomize here). cc @TheSpiritXIII

Reply:
The alternative (or actually doing both ideas might work) would be to have official "dev" registry we will push ALWAYS and register the version here somehow (maybe "main" tag after all). Not tested, but it will be builded like any other image. Then we can promote to official prod registry and tag on the final release.